utils/ocm_extractor.py

```diff
@@ -0,0 +1,105 @@
+import hashlib
+import json
+import os
+import shutil
+from numbers import Number
+from typing import Callable, Dict, List, Optional
+import pandas as pd
+import git
+
+from utils.logging_utils import log
+
+
+def ocm_extractor():
+    # Dataframes
+    f_stations = pd.DataFrame()
+    f_connection = pd.DataFrame()
+    f_countries = pd.DataFrame()
+    f_operators = pd.DataFrame()
+    f_stations_connections = pd.DataFrame()
+
+    # directory
+    dirname = os.path.dirname(__file__)
+    data_path = os.path.join(dirname, '..', '..', '..')
+
+    # function to delete the repo
+    def delete_directory(direct):
+        # getting the folder path from the user
+        folder_path = direct
+
+        try:
+            shutil.rmtree(folder_path)
+        except OSError as e:
+            log.warn("Delete directory: %s - %s." % (e.filename, e.strerror))
+
+    directory_to_delete = os.path.join(data_path, 'ocm-export')
+    # Delete the repo before cloning
+    delete_directory(directory_to_delete)
+
+    # get data from ocm Repo
+    try:
+        git.Git("/test").clone("https://github.com/openchargemap/ocm-export")
+    except:
+        print("no need to clone")
+    rootdir = 'ocm-export/data/DE'
```

Comment:
root dir should also be relative directory and created using os.path.join() as the other directories